docs/editor/extension-gallery.md

```diff
@@ -23,31 +23,32 @@ This will show you a list of the most popular VS Code extensions on the [VS Code
 
 ![popular extensions](images/extension-gallery/extensions-popular.png)
 
-Each extension in the list includes a brief description, the publisher, the download count and a five star rating. You can click on the extension item to display the extension's [VS Code Marketplace](https://marketplace.visualstudio.com/VSCode) page where you can learn more.
+Each extension in the list includes a brief description, the publisher, the download count and a five star rating. You can click on the extension item to display the extension's details page where you can learn more.
 
 > **Note:** If your computer's Internet access goes through a proxy server, you will need to configure the proxy server. See [Proxy server support](/docs/setup/network.md#proxy-server-support) for details.
 
 ### Install an extension
 
-Click the **Install** button and after a successful install, you'll see an **Reload** button which will prompt you to restart VS Code to enable the new extension.
+Click the **Install** button and after a successful install, you'll see a **Reload** button. Clicking on this will restart VS Code to enable the new extension.
 
 ### Extension details
 
-In the extension Details window, you can read the extension's README as well as review the extension's:
+On the extension details page, you can read the extension's README as well as review the extension's:
 
 * **Contributions** - The extension's additions to VS Code such as settings, commands and keyboard shortcuts, language grammars, debugger, etc.
 * **Changelog** - The extension repository CHANGELOG if available.
 * **Dependencies** - Lists if the extension depends on any other extensions.
 
 ![extension contributions](images/extension-gallery/extension-contributions.png)
 
-If an extension is an Extension Pack, the **Dependencies** section will display which extensions will be installed. [Extension Packs](/docs/extensionAPI/extension-manifest.md#extension-packs) bundle separate extensions together so they can be easily installed at one time.
+[Extension Packs](/docs/extensionAPI/extension-manifest.md#extension-packs) bundle separate extensions together so they can be easily installed at one time. If an extension is an Extension Pack, the **Dependencies** and **Extension Pack** sections will display which extensions will be installed when you install the pack.
```

Comment:
@gregvanl here I'm not sure of a good way to say that using `Dependencies` to specify a pack should be considered deprecated, as it causes unwanted behavior when disabling a component extension. `Extension Pack` is a new section this release so everything out there is going to be using `Dependencies` for some time now.

Reply:
I didn't think the Dependencies list is deprecated. Some extensions really do have dependencies on other extensions (for example most Azure extensions depend on the Azure Account extension). Might be good to describe the difference here.

Reply:
Let's not touch this section for now. Once 1.26 is out in the wild for a month and all extension packs have updated to the new model, we can come back and update this.